Gruntfile.js

```diff
@@ -92,6 +92,7 @@ module.exports = function(grunt) {
   // on each recompile
   var updateTsFiles = function() {
     tsFiles = grunt.file.read("src/reference.ts")
+      .replace(/\r\n/g, "\n")
```

Comment:
Yes this (grunt.util.linefeed) is a better solution than my first attempt. The key thing I want to achieve is to get the same behaviour from the grunt tasks under Windows as elsewhere, including
- gruntfile parses and can execute;
- outputs of compiles give identical plottable.js and plottable.d.ts
- tests execute and give the same results

At the moment, I get blank lines through plottable.d.ts. This is caused by the sed task removing the private declarations - it doesn't remove the line-end properly. This can probably be fixed by substituting grunt.util.linefeed into these statements where needed:
```javascript
   var prefixMatch = "\\n *(function |var |static )?";
    var varNameMatch = "[^(:;]*(\\([^)]*\\))?"; // catch function args too
    var nestedBraceMatch = ": \\{[^{}]*\\}";
    var typeNameMatch = ": [^;]*";
    var finalMatch = "((" + nestedBraceMatch + ")|(" + typeNameMatch + "))?\\n?;";
    var jsdocInit = "\\n *\\/\\*\\* *\\n";
    var jsdocMid = "( *\\*[^\\n]*\\n)+";
    var jsdocEnd = " *\\*\\/ *";
    var jsdoc = "(" + jsdocInit + jsdocMid + jsdocEnd + ")?";
```

(i will try then report back)

Also, currently I get a few test fails just fetching the current develop branch and executing test - am I right to think this shouldn't happen? e.g.
```

  1) NumericAxis draws ticks correctly (vertical):
     AssertionError: tick label is centered on mark: expected 3501.5 to be close to 3503 +/- 1
```